docs/source/admin/upgrading.rst

```diff
@@ -27,51 +27,50 @@ Notify your users
 
 If you are using the default configuration where ``configurable-http-proxy``
 is managed by JupyterHub, your users will see service disruption during
-the upgrade process. You should notify them, and pick a time to do the
+the upgrade process. You will need notify them, and pick a time to do the
```

Comment:
Hi, 
I think this sounds better as "you will need to notify them", instead of "you will need notify them"